eis_toolkit/exceptions.py

```diff
@@ -88,3 +88,19 @@ class InvalidModelException(Exception):
 
 class InvalidDatasetException(Exception):
     """Exception error class when the dataset is null."""
+
+
+class CNNException(Exception):
+    """Exception throws when something is invalid in the cnn."""
+
+
+class CNNRunningParameterException(Exception):
+    """Exception throws when running parameters are wrong."""
+
+
+class InvalidCrossValidationSelected(Exception):
+    """Exception throws when running parameters are wrong."""
+
+
+class InvalidNumberOfSplit(Exception):
+    """Exception throws when running parameters are wrong."""
```

Comment:
I don't think we need any of these exception types to be added. The ones already should suffice. Please take a look at what other prediction functions are currently using in EIS Toolkit.

Reply:
ok

Reply:
I left the two for the cv because I think are good! Others are ok!